### PR TITLE
pAI chassis: Protogen Dog

### PR DESCRIPTION
**Pull request by OverDriveZ**
## About The Pull Request
I'd like to request the addition of these sprites in case you deem them worthy enough to add in! 
Here comes the protogen pAI chassis!
Here's a preview of what it could be:

protodog
## Changelog
:cl:
add: New pAI chassis
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/pai/pai.dm [hunk 1]
```diff
@@ -57,7 +57,8 @@
 		"Catslug" = "catslug",
 		"Car" = "car",
 		"Type One" = "typeone",
-		"Type Thirteen" = "13"
+		"Type Thirteen" = "13",
+		"Protogen Dog" = "pai-protodog"
```
Comment (Kashargul) [inline code/modules/mob/living/silicon/pai/pai.dm:61]: Missing //CHOMPAdd 
comment

File: code/modules/mob/living/silicon/pai/pai_vr.dm [hunk 2]
```diff
@@ -17,7 +17,8 @@
 		"pai-diredog",
 		"pai-horse_lune",
 		"pai-horse_soleil",
-		"pai-pdragon"
+		"pai-pdragon",
+		"pai-protodog"
```
Comment (Kashargul) [inline code/modules/mob/living/silicon/pai/pai_vr.dm:21]: Same as before.

File: code/modules/mob/living/silicon/pai/pai_vr.dm [hunk 3]
```diff
@@ -55,7 +56,8 @@
...
```
Comment (Kashargul) [inline code/modules/mob/living/silicon/pai/pai_vr.dm:60]: Same as before.

**Review by Kashargul — CHANGES_REQUESTED** [review]
Why is the dll file touched?

**Comment by OverDriveZ** [comment]
Forgot to connect my visual code remote, the branch I worked on is outdated, I'll open a new PR without these issues

**Pull request closed without merging**